lib/js/src/rpc/messages/RegisterAppInterfaceResponse.js

```diff
@@ -44,6 +44,7 @@ import { PrerecordedSpeech } from '../enums/PrerecordedSpeech.js';
 import { FunctionID } from '../enums/FunctionID.js';
 import { AudioPassThruCapabilities } from '../structs/AudioPassThruCapabilities.js';
 import { VrCapabilities } from '../enums/VrCapabilities.js';
+import { HMICapabilities } from '../structs/HMICapabilities.js';
```

Comment:
getHMICapabilities and setHMICapabilities both use this. I will have to check the unit tests branch for this being tested properly.

Reply:
sdl_java_suite/android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/responses/RegisterAppInterfaceResponseTest.java 
and
sdl_javascript_suite/tests/node/rpc/messages/RegisterAppInterfaceResponseTests.js

Don't test hmi capabilities.